### DISCUSSION TITLE
MFA for Admin

### DISCUSSION THREAD

**Discussion by khrome83** [pocketbase/pocketbase#3381] [Q&A]
The admin account currently does not have MFA on it. 

I am tempted to put the whole app behind a wireguard tunnel, but that makes it more complicated. 

Can I suggest adding MFA or alternate login mechanism for admin?

This seems reasonable easy, outside of the UI elements that would need to be built.

https://medium.com/@kittipat_1413/implementing-two-factor-authentication-2fa-with-totp-in-golang-1a3fd0dd7662

**Comment by khrome83** [comment]
I would also like to enable forcing MFA for new admins that are added. So they have to set it up upon first login. But i can see this being a separate feature further down the road.

**Comment by enriquebarco** [comment]
what happened with this? was 2fa ever added? I do not see support

**Reply by chnfyi** [reply]
MFA is implemented as part of v0.23.0rc, you can take a look to the changelog [here](https://github.com/pocketbase/pocketbase/blob/develop/CHANGELOG.md#v0230-rc) and try it.

**Reply by enriquebarco** [reply]
Thanks!

**Reply by levino** [reply]
~I am sorry, but I do not understand. Is it possible now? I am on 0.24.3 and I am not seeing such option.~ I found it:
![Screenshot from 2025-01-29 13-30-43](https://github.com/user-attachments/assets/9a9fa262-265b-4aa2-a8d8-0414a4251686)
